git-cliff/src/lib.rs

```diff
@@ -110,7 +121,7 @@ fn process_repository<'a>(
 				})
 				.unwrap_or_default();
 
-			skip || !ignore
+			skip || (count && !ignore)
```

Comment:
I'm a bit hesitate if we should use `skip || (count && !ignore)` or `skip || count || !ignore`.

They are the same if ignore is empty. But the formal means ignore takes priority to count, and the latter means count takes priority to ignore.

Maybe not a logically better choice, so I leave it as is now.

Reply:
No. It's not about priority, but if more strict or looser.